tests/arch/arc/arc_dsp_sharing/testcase.yaml

```diff
@@ -0,0 +1,8 @@
+tests:
+  kernel.dsp_sharing.generic.arc:
+    toolchain_allow: arcmwdt
+    filter: CONFIG_ISA_ARCV2 and CONFIG_ARC_HAS_DSP
+    slow: true
+    tags: dsp arc
+    timeout: 600
+    min_ram: 16
```

Comment:
This looks like a copy-paste from `tests/kernel/fpu_sharing/generic/testcase.yaml`. :)
If it's so and we haven't do some memory estimations for this test - let's simply drop it.